builtin/providers/aws/resource_aws_autoscaling_group.go

```diff
@@ -446,7 +446,6 @@ func resourceAwsAutoscalingGroupRead(d *schema.ResourceData, meta interface{}) e
 	d.Set("health_check_type", g.HealthCheckType)
 	d.Set("launch_configuration", g.LaunchConfigurationName)
 	d.Set("load_balancers", flattenStringList(g.LoadBalancerNames))
-	d.Set("target_group_arns", flattenStringList(g.TargetGroupARNs))
```

Comment:
This was duplicated here https://github.com/hashicorp/terraform/blob/1f13ff22a3f4961782e3c4793510a4d966169e9a/builtin/providers/aws/resource_aws_autoscaling_group.go#L453 so I removed the redundant piece